avocado/utils/configure_network.py

```diff
@@ -132,7 +132,7 @@ def set_mtu_host(self, interface, mtu):
             cmd = "ip add show %s" % interface
             mtuvalue = process.system_output(cmd, shell=True).decode("utf-8") \
                                                              .split()[4]
-            if mtuvalue == mtu:
+            if str(mtuvalue) == str(mtu):
```

Comment:
`mtu` is a function parameter so why we need to type cast may be it can be doc string